backend/pkg/modules/internal/fuzzer/tools/tools.go

```diff
@@ -120,3 +135,15 @@ func DumpHTTPFuzzParam(params restapi.FuzzTargetParams) string {
 	ret = ret + "}"
 	return ret
 }
+
+//nolint:varnamelen
+func TrimLeftChars(s string, n int) string {
```

Comment:
please add unit test.
also, not sure you need to loop over s, why not just do
```
	if n > len(s) {
		return ""
	}
	return s[n:]
```

Reply:
It was an implementation found somewhere. But following your remarks I see that "strings.TrimPrefix" exists and do exactly what I need. I will remove this "trimLeftChars" function.
Thanks.